package.json

```diff
@@ -17,9 +17,12 @@
   "bugs": {
     "url": "https://github.com/peterianmuller/quiz-library/issues"
   },
-  "homepage": "https://github.com/peterianmuller/quiz-library#readme",
-  "dependencies": {},
+  "homepage": "https://github.com/peterianmuller/quiz-library#readme",  
```

Comment:
blocking: this is no longer true; pim => 16b; same for url attr above. let's update it throughout.
Could be a separate pr if you want.